front/src/common/redux/set-password/set.password.api.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+export async function setPassswordApi (request) {
+
+    const url = `${process.env.REACT_APP_SET_PASSWORD_ROUT}${request.data.token}`;
+    const headers = { headers: { 'Content-Type': 'application/json' } };
+   
+    return axios
+        .put(url, request.data, headers)
+        .then(response => response)
```

Comment:
Please, return only data from API function, don't return the full response, saga should not know anything about HTTP requests

Reply:
Fixed